firmware/boron-ins-fsm/src/BraveSensorProductionFirmware.ino

```diff
@@ -11,10 +11,8 @@
 #include "tpl5010watchdog.h"
 
 #define DEBUG_LEVEL            LOG_LEVEL_INFO
-#define BRAVE_FIRMWARE_VERSION 10070  // see versioning notes in the readme
-#define BRAVE_PRODUCT_ID       14807  // 14807 = beta units, 15479 = production units
+#define BRAVE_FIRMWARE_VERSION 12  // see versioning notes in the readme
 
-PRODUCT_ID(BRAVE_PRODUCT_ID);             // you get this number off the particle console, see readme for instructions
```

Comment:
Nope! One of the only breaking changes moving from device OS 3.3.1 --> device OS 4.2.0 is the removal of PRODUCT_ID. Makes it way simpler to compile firmware between dev and prod, though, so that's cool!